ui/src/components/SearchControls/components/SearchAggregations/SearchAggregationsCards.js

```diff
@@ -21,17 +21,18 @@ export default class SearchAggregationsCards extends Component {
       getChildAggCmps
     ) => {
       const childAggCmps = getChildAggCmps(bucket);
+      const key = bucket.key_as_string ? bucket.key_as_string : bucket.key;
```

Comment:
What about changing the values `true/false` to `Open Access/Closed Access`?

Reply:
do you think it might have any significant performance influence for this case if we index string/keyword instead of boolean?

Reply:
You could just add a label for the UI for true/false, right?